src/js/app.js

```diff
@@ -144,20 +147,22 @@ angular.module('blocktrail.wallet').run(
         $rootScope.isSamsung = !!$cordovaDevice.getModel().toLowerCase().match(/samsung/);
 
         $rootScope.changeLanguage = function(language) {
-            $log.debug('changeLanguage: ' + language);
-            settingsService.language = language || blocktrailLocalisation.preferredAvailableLanguage() || CONFIG.FALLBACK_LANGUAGE || 'en';
```

Comment:
we used to set `settingsService.language` here, now when you change langauge i dont think its saved at all?